tests/cpp/aot/llvm/field_aot_test.cpp

```diff
@@ -33,12 +33,9 @@ void run_field_tests(aot::Module *mod,
   aot::Kernel *k_check_activate_pointer_fields =
       mod->get_kernel("check_activate_pointer_fields");
 
-  // Initialize Fields
-  aot::Field *field_x = mod->get_field("0" /*snode_tree_id*/);
-  aot::Field *field_y = mod->get_field("0" /*snode_tree_id*/);
-
-  finalize_aot_field(mod, field_x, result_buffer);
-  finalize_aot_field(mod, field_y, result_buffer);
+  // Initialize SNodeTree
+  aot::Field *snode_tree_0 = mod->get_field("0" /*snode_tree_id*/);
```

Comment:
OOC: is it possible to rename `get_field` to `get_snode_tree` to make it easier to understand?  :P